source/_layouts/documentation.blade.php

```diff
@@ -1,62 +1,71 @@
 @extends('_layouts.master')
 
 @section('body')
-<nav class="navbar navbar-brand">
-    <div class="container">
-        <div class="navbar-content">
-            <div>
-                <a class="link-plain text-xxl flex-y-center" href="{{ $page->asset_prefix }}/">
-                    <img src="{{ $page->asset_prefix }}/img/jigsaw-logo.svg" alt="Jigsaw logo" class="logo-icon" width="65">
-                    <img src="{{ $page->asset_prefix }}/img/jigsaw-type.svg" alt="Jigsaw logo" class="logo-type" width="130">
-                </a>
-            </div>
-            <div class="docsearch navbar-buttons">
-                <input id="docsearch" class="docsearch__input" type="text" name="docsearch" value=""/>
-                <span class="docsearch__icon"></span>
-            </div>
+<header class="w-full bg-white absolute z-10 shadow-md">
+    <nav class="flex items-center justify-center py-4 container-content" aria-role="navigation">
+        <a href="/" class="flex items-center lg:w-1/3" title="Jigsaw by Tighten">
+            <img src="{{ $page->asset_prefix }}/assets/img/jigsaw-logo.svg" alt="Jigsaw logo"
```

Comment:
I killed `asset_prefix` on `develop` because it's not doing anything

Reply:
Removed references in `navigation.vue` and `navigation.blade.php` as well. Thanks!